src/Azure.Functions.Cli/Helpers/WorkerRuntimeLanguageHelper.cs

```diff
@@ -39,7 +41,8 @@ public static class WorkerRuntimeLanguageHelper
             { WorkerRuntime.dotnet, Constants.Languages.CSharp },
             { WorkerRuntime.node, Constants.Languages.JavaScript },
             { WorkerRuntime.python, Constants.Languages.Python },
-            { WorkerRuntime.powershell, Constants.Languages.Powershell }
+            { WorkerRuntime.powershell, Constants.Languages.Powershell },
+            { WorkerRuntime.custom, Constants.Languages.Custom },
```

Comment:
Can you confirm that this will also set `FUNCTIONS_WORKER_RUNTIME` to `custom`?

Reply:
You mean in local.settings.json, right ? Yeah it had `FUNCTIONS_WORKER_RUNTIME` set to `custom`